src/main/java/Ui.java

```diff
@@ -0,0 +1,31 @@
+//package duke;
+/**
+ * interact with user
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    /**
+     * greet user
+     */
```

Comment:
Perhaps you could replace the JavaDoc method header comment for this method as "Greets user"? This was mentioned in the guidelines for Java coding standard which mentioned that "In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)".